src/components/checkBoxSelection/checkBoxSelection.scss

```diff
@@ -36,17 +28,62 @@
   margin-top: 20px;
   display: none;
 }
-
 .card-settings {
-  height: 600px;
+  height: 350px;
+  margin-left: 159px;
   overflow-y: scroll;
-  background-color: #b5cfe1;
-  width: 350px;
-  padding-left: 20px;
-  box-shadow: 10px 10px 55px 0px black;
+  background-color:#b5cfe1 ;
+  width: 600px;
+  box-shadow: 10px 10px 55px 0px #000000;
 }
-
 .slider {
   margin-top: 10px;
   margin-bottom: 10px;
 }
+.opt {
+  margin-left: 350px;
+  padding-bottom: 800px;
+}
+.home-display-class {
+  display: flex;
+}
+.choose {
+  width: 50%;
+}
+.display {
+  width: 50%;
+}
+
+.optimize-button{
+  margin-top: 12%;
+  margin-top: 19%;
+  text-align: center;
+  display: inline-block;
+  justify-items: center;
+  width:100px;
+  height: 70px;
+  border-radius: 35px;
+  background-color: #b5cfe1;
+  align-items: center;
+}
+
+.download-Button {
+  text-align: center;
+  display: inline-grid;
+  justify-items: center;
+  width: 100px;
+  height: 70px;
+  border-radius: 35px;
```

Comment:
again, high choice of radius, play with this amount and see if you can find something more subtle

Reply:
yeah, im trying to improve the design in this pr as well. the first thing a user sees is the ui before using the product, and if users dont want to use it then its useless. Please refer to this link for any design ambiguity https://svgomg.firebaseapp.com/